firmware/applications/SignBuddy/task_config.h

```diff
@@ -5,10 +5,12 @@
 #define SENSORS_TASK_PRIORITY    (tskIDLE_PRIORITY + 3)
 #define TSC_TASK_PRIORITY        (tskIDLE_PRIORITY + 4)
 #define FLEX_TASK_PRIORITY       (tskIDLE_PRIORITY + 4)
+#define IMU_TASK_PRIORITY        (tskIDLE_PRIORITY + 4)
 
-#define SENSORS_STACK_SIZE       512
+#define SENSORS_STACK_SIZE       256
 #define TSC_STACK_SIZE           256
 #define FLEX_STACK_SIZE          256
+#define IMU_STACK_SIZE           400
```

Comment:
Reduce back to 256 if that still works

Reply:
Thanks for the suggestions. I will work on all these and fix them. Most likely tomorrow. I think we may need to keep it at 300 or something because initially it still takes about 260ish.